klippy/extras/thermocouple.py

```diff
@@ -0,0 +1,85 @@
+# Temperature measurements with thermocouples. Only support for Type K for now.
+# Cold junction compensation
```

Comment:
Please start the file with a one line comment, then an empty comment line, then the copyright statement, then an empty comment line, then the license statement.  Feel free to add any number of comments later in the file, but I'd prefer to keep the top copyright statement uniform across files.